dart/lib/src/sentry_options.dart

```diff
@@ -355,6 +355,9 @@ class SentryOptions {
   late SentryStackTraceFactory stackTraceFactory =
       SentryStackTraceFactory(this);
 
+  /// Automatically attaches a screenshot when capturing an error or exception.
+  bool attachScreenshot = false;
```

Comment:
This should go under the sentry flutter options

Reply:
True, we don't need this in the dart part